src/cc/CMakeLists.txt

```diff
@@ -47,11 +47,11 @@ set(bcc_common_sources ${bcc_common_sources} bpf_module_rw_engine_disabled.cc)
 endif()
 
 set(bcc_table_sources table_storage.cc shared_table.cc bpffs_table.cc json_map_decl_visitor.cc)
-set(bcc_util_sources ns_guard.cc common.cc)
-set(bcc_sym_sources bcc_syms.cc bcc_elf.c bcc_perf_map.c bcc_proc.c)
+set(bcc_util_sources common.cc)
+set(bcc_sym_sources bcc_syms.cc bcc_proc.h bcc_elf.c bcc_perf_map.c bcc_proc.c)
```

Comment:
We don't put header files here (`bcc_sym_sources`).

Reply:
will figure out why i thought this was necessary and remove / adjust